doc/contributing/docs/sphinx_warnings.rst

```diff
@@ -0,0 +1,194 @@
+Sphinx-build warnings reference
+-------------------------------
+
+This document will guide you through the possible warnings raised by Sphinx engine
+while building the docs.
+
+Warnings and solutions list
+===========================
+
+Below we cite a list with the most frequent warnings and the ways of solutions.
+
+Bullet list ends without a blank line; unexpected unindent
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Similar warning:** Block quote ends without a blank line; unexpected unindent
+
+**Example:**
+
+..  code-block:: rst
+
+    *   The last point of bullet list
+    This text should start after a blank line
+
+**Solution:**
+
+..  code-block:: rst
+
+    *   The last point of bullet list
+
+    This text should start after a blank line
+
+Could not lex literal_block as "...". Highlighting skipped
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Example:**
+
+**Solution:**
+
+Duplicate explicit target name: "..."
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Example:**
+
+..  code-block:: rst
+
+    * `Install <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_
+      ``git``, a version control system.
+
+    * `Install <https://linuxize.com/post/how-to-unzip-files-in-linux/>`_
+      the ``unzip`` utility.
+
+**Solution:**
+
+Doc-builder doesn't like when we call different targets the same names.
+Sphinx developers `recommend <https://github.com/sphinx-doc/sphinx/issues/3921>`_
+using double underlines ``__`` in such cases to avoid this warning.
```

Comment:
It's better not to refer to computers or programs as sentient beings. They cannot “like” things. Also, what's this “doc-builder”? I only know the `sphinx-builder` app.